src/info.js

```diff
@@ -0,0 +1,145 @@
+    renderArg(into, typeInfo, options);
+    renderDescription(into, options, typeInfo.argDef.description);
+    return into;
+  } else if (kind === 'NamedType' && step === 0 &&
+             typeInfo.type && typeInfo.type.description) {
+    const into = document.createElement('div');
+    renderType(into, typeInfo, options, typeInfo.type);
+    renderDescription(into, options, typeInfo.type.description);
+    return into;
+  }
+});
+
+function renderDescription(into, options, description) {
+  if (description) {
+    const descriptionDiv = document.createElement('div');
+    descriptionDiv.className = 'info-description';
+    if (options.renderDescription) {
+      descriptionDiv.innerHTML = options.renderDescription(description);
+    } else {
+      descriptionDiv.textContent = description;
+    }
+    into.appendChild(descriptionDiv);
+  }
+}
+
+function renderField(into, typeInfo, options) {
+  renderQualifiedField(into, typeInfo, options);
+  renderTypeAnnotation(into, typeInfo, options, typeInfo.type);
+}
+
+function renderQualifiedField(into, typeInfo, options) {
+  const fieldName = typeInfo.fieldDef.name;
+  if (fieldName.slice(0, 2) !== '__') {
+    renderType(into, typeInfo, options, typeInfo.parentType);
+    text(into, '.');
+  }
+  text(into, fieldName, 'field-name', options, getFieldReference(typeInfo));
+}
+
+function renderDirective(into, typeInfo, options) {
+  const name = '@' + typeInfo.directiveDef.name;
+  text(into, name, 'directive-name', options, getDirectiveReference(typeInfo));
+}
+
+function renderArg(into, typeInfo, options) {
+  if (typeInfo.directiveDef) {
+    renderDirective(into, typeInfo, options);
+  } else if (typeInfo.fieldDef) {
+    renderQualifiedField(into, typeInfo, options);
+  }
+
+  const name = typeInfo.argDef.name;
+  text(into, '(');
+  text(into, name, 'arg-name', options, getArgumentReference(typeInfo));
+  renderTypeAnnotation(into, typeInfo, options, typeInfo.inputType);
+  text(into, ')');
+}
+
+function renderTypeAnnotation(into, typeInfo, options, t) {
+  text(into, ': ');
+  renderType(into, typeInfo, options, t);
+}
+
+function renderType(into, typeInfo, options, t) {
+  if (t instanceof GraphQLNonNull) {
+    renderType(into, typeInfo, options, t.ofType);
+    text(into, '!');
+  } else if (t instanceof GraphQLList) {
+    text(into, '[');
+    renderType(into, typeInfo, options, t.ofType);
+    text(into, ']');
+  } else {
+    text(into, t.name, 'type-name', options, getTypeReference(typeInfo, t));
+  }
+}
+
+function text(into, content, className, options, ref) {
+  if (className) {
+    const node = document.createElement(options.onClick ? 'a' : 'span');
+    if (options.onClick) {
+      // Providing a href forces proper a tag behavior, though we don't actually
+      // want clicking the node to navigate anywhere.
+      node.href = 'javascript:void 0'; // eslint-disable-line no-script-url
+      node.addEventListener('click', function (e) {
```

Comment:
It will get cleaned up by the GC when the containing DOM is removed. There's no circular reference within the handler, so it should be safe. IE6-8 suffered from some event handler memory leak bugs, but I think it's safe to ignore them.